### DISCUSSION TITLE
Darkmoon Card: Crusade

### DISCUSSION THREAD

**Discussion by Deligted** [chromiecraft/chromiecraft#5869] [General]
Darkmoon Card: Crusade trinket not working as it should for Retribution Paladins :

AP buff

Auto-attack
Seal of Command swings (no other Seal damage)
Crusader Strike

SP buff

Judgement
First tic of Consecration (one stack per mob affected)
Exorcism
Holy Wrath (one stack per mob affected)
Repentance
Hammer of Justice
Arcane Torrent (one stack per mob affected)

Source - https://www.wowhead.com/wotlk/item=31856/darkmoon-card-crusade#comments

**Comment by r-o-b-o-t-o** [comment]
Please use the Issues tab to report bugs: https://github.com/chromiecraft/chromiecraft/issues/new/choose
